corehq/apps/app_manager/detail_screen.py

```diff
@@ -352,8 +352,39 @@ def variables(self):
         return variables
 
 
+@register_format_type('conditional-enum')
+class ConditionalEnum(Enum):
+    @property
+    def sort_node(self):
+        node = super(ConditionalEnum, self).sort_node
+        if node:
+            node.order = "-2"
+            node.direction = "ascending"
```

Comment:
Thank You Simon. Removed the order and direction assignment. So now the sort node is added without any order and direction. Working well now.